label_studio/io_storages/azure_blob/models.py

```diff
@@ -45,21 +45,38 @@ class AzureBlobStorageMixin(models.Model):
     account_key = models.TextField(
         _('account_key'), null=True, blank=True,
         help_text='Azure Blob account key')
+    sas_token = models.TextField(
+        _('sas_token'), null=True, blank=True,
+        help_text='Azure Blob account key')
 
     def get_account_name(self):
         return str(self.account_name) if self.account_name else get_env('AZURE_BLOB_ACCOUNT_NAME')
 
     def get_account_key(self):
         return str(self.account_key) if self.account_key else get_env('AZURE_BLOB_ACCOUNT_KEY')
 
+    def get_sas_token(self):
+        return str(self.sas_token) if self.sas_token else get_env('AZURE_BLOB_SAS_TOKEN')
+
+    def get_container(self):
```

Comment:
this method duplicates [another implementation of `get_container()`](https://github.com/heartexlabs/label-studio/blob/master/label_studio/io_storages/azure_blob/models.py#L63). Not sure it will work correctly when returning strings instead of `ContainerClient` instance.

Reply:
Removed this function